src/common/schemas/PatchRetentionRule.yml

```
@@ -1,19 +1,29 @@
-type: object
-description: Updates to a rule to expire or retain data.
-properties:
-  type:
-    type: string
-    default: expire
-    enum:
-      - expire
-  everySeconds:
-    type: integer
-    format: int64
-    description: Duration in seconds for how long data will be kept in the database. 0 means infinite.
-    example: 86400
-    minimum: 0
-  shardGroupDurationSeconds:
-    type: integer
-    format: int64
-    description: Shard duration measured in seconds.
-required: [type]
+  type: object
+  properties:
+    type:
+      type: string
+      default: expire
+      enum:
+        - expire
+    everySeconds:
+      type: integer
+      format: int64
+      description: |
+        The duration in seconds for how long data will be kept in the database.
+        The default duration is 2592000 (30 days).
+        0 represents infinite retention.
+      example: 86400
+      default: 2592000
+      minimum: 0
+    shardGroupDurationSeconds:
+      type: integer
+      format: int64
+      description: |
+        The shard group duration.
+        The duration or interval (in seconds) that each shard group covers.
+        #### InfluxDB Cloud
+        - Does not use `shardGroupDurationsSeconds`.
+        #### InfluxDB OSS
+        - Default value depends on the
+        [bucket retention period]({{% INFLUXDB_DOCS_URL %}}/v2.3/reference/internals/shards/#shard-group-duration).
```

- I'm not sure what "duration or interval" means here.
- What does "Default value depends on..." mean?

```suggestion
        The shard group duration.
        The number of seconds that each shard group covers.
        
        #### InfluxDB Cloud
        
        - Doesn't use `shardGroupDurationsSeconds`.
        
        #### InfluxDB OSS
        
        - Default value depends on the
        [bucket retention period]({{% INFLUXDB_DOCS_URL %}}/v2.3/reference/internals/shards/#shard-group-duration).
```

---

1.) Shards contain an allotted amount of data based off a given time range / duration. If a shard group duration is set to 30days, then each shard will only contain 30days worth of data.

2.) Default value depends on bucket retention period. i moved `bucket retention period` to a new line to try to shorten the line length

---

ill add add a related guide and link out to the shard glossary for additional information